specification/authorization/resource-manager/Microsoft.Authorization/preview/2020-08-01-preview/authorization-RoleAssignmentsCalls.json

```diff
@@ -658,13 +658,7 @@
             "User",
             "Group",
             "ServicePrincipal",
-            "Unknown",
-            "DirectoryRoleTemplate",
-            "ForeignGroup",
-            "Application",
-            "MSI",
-            "DirectoryObjectOrGroup",
-            "Everyone"
```

Comment:
These shouldn't be used by users
Linting since other teams requested it as well.

Reply:
Same for all other similar changes

Reply:
@amanswaika

Reply:
resolved offline, closing